wiki/osu!supporter/en.md

```diff
@@ -164,6 +165,7 @@ The giver can also hide this event from their own profile by ticking `Hide all o
 
 Additionally, both the giver and the recipient will receive email notifications mentioning the purchase.
 
-## References
+## Notes
 
 [^pending-beatmaps-ref]: [Forum post by peppy (2021-09-14) in "Increase the number of pending beatmap slots"](https://osu.ppy.sh/community/forums/posts/8294132)
+[^team-size-note]: Each supporter will add 4 slots to their team's base size of 8, up to a cap of 256.
```

Comment:
why is this note needed 🤔
isn't it just restating the formula? the other table rows don't have that

Reply:
The difference is that the other limits aren't really "Dynamic". The only similar one being Pending beatmaps. However this is not a limit individual to the user, so I just wanted to make it clear that the limit depends on more supporters, other than the user.